lte/gateway/c/session_manager/StoredState.h

```diff
@@ -201,6 +201,8 @@ struct SessionCreditUpdateCriteria {
   GrantTrackingType grant_tracking_type;
   // Do not mark REPORTING buckets, but do mark REPORTED
   std::unordered_map<Bucket, uint64_t> bucket_deltas;
+
+  bool deleted;
```

Comment:
btw i forgot to ask: even that this field is only use for deletion, so the rule is deleted and there is nothing to store after this happens. Not sure if what i am saying is right, but what happen to this field when the rule goes into the store and its value is false? next time we umarshal that value will not be there, and we will be using the default value (that coincidentally ix false). Is that correct?

I so, then maybe we need it to add it to hte store just in case in the future we dont delete the rule immediatley or something like this. Finding where the true became false may be an issue